x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/openai/embeddings/OpenAiEmbeddingsTaskSettings.java

```diff
@@ -30,53 +27,25 @@
 /**
  * Defines the task settings for the openai service.
  *
- * @param modelId the id of the model to use in the requests to openai
- * @param user an optional unique identifier representing the end-user, which can help OpenAI to monitor and detect abuse
- *             <a href="https://platform.openai.com/docs/api-reference/embeddings/create">see the openai docs for more details</a>
+ * User is an optional unique identifier representing the end-user, which can help OpenAI to monitor and detect abuse
+ *  <a href="https://platform.openai.com/docs/api-reference/embeddings/create">see the openai docs for more details</a>
  */
-public record OpenAiEmbeddingsTaskSettings(String modelId, @Nullable String user) implements TaskSettings {
+public class OpenAiEmbeddingsTaskSettings implements TaskSettings {
 
     public static final String NAME = "openai_embeddings_task_settings";
     public static final String OLD_MODEL_ID_FIELD = "model";
-    public static final String MODEL_ID = "model_id";
+
```

Comment:
Can we remove `OLD_MODEL_ID_FIELD` or is that referenced in other places?

Reply:
In one other place, I've moved it there